Dockerfile

```diff
@@ -24,9 +24,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends gconf-service l
   && apt-get update \
   && apt-get install -y google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst ttf-freefont \
     --no-install-recommends \
-  && npm install -g yarn@$YARN_VERSION \
-  && npm install -g s3-cli \
-  && chmod +x /usr/local/lib/node_modules/yarn/bin/yarn.js
```

Comment:
Our Docker image installs the same version of Yarn, causing a conflict. I'm removing our Yarn stuff here, but the better fix would probably be to remove our dependency on this Docker image.

Reply:
switching to nvm would be handy sure, but not a requirement to get things moving today.